ReportPortal.SpecFlowPlugin/ReportPortalAddin.cs

```diff
@@ -36,6 +36,12 @@ internal static void SetFeatureTestReporter(FeatureContext context, ITestReporte
             FeatureThreadCount[context.FeatureInfo] = 1;
         }
 
+        internal static void RemoveFeatureTestReporter(FeatureContext context, ITestReporter reporter)
+        {
+            FeatureTestReporters.TryRemove(context.FeatureInfo, out reporter);
+            FeatureThreadCount[context.FeatureInfo] -= 1;
```

Comment:
@nvborisenko Is there a reason to decrease the `FeatureThreadCount` instead of just removing it from the dictionary as we do in `FeatureTestReporters`? I thought these clean up changes were done to remove extra items from the dictionary so the retrieval of the item by its key will be faster.

Reply:
Adjusted.